docker/official/remco/templates/artifact-repositories-private.yaml

```diff
@@ -0,0 +1,21 @@
+{% set pluginProviderBase = "/rundeck/repository/artifacts/provider" %}
+
+repositories:
+  - repositoryName: official
+    enabled: true
+    type: HTTP
+    owner: RUNDECK
+
+  {%- macro plugin_provider(provider) %}
```

Comment:
Should this be `artifact_provider`, or does it not need to link up in any way to the same in the rundeck-config template?

Reply:
it is just the name because I use `{% set pluginProviderBase = "/rundeck/repository/artifacts/provider" %}`

I will change the name to avoid confusions

Reply:
done